inference-engine/src/preprocessing/ie_preprocess_gapi_kernels.cpp

```diff
@@ -468,15 +468,86 @@ struct type_to_type {};
 template <typename typelist>
 struct type_dispatch_impl;
 
+//FIXME: add test for type_dispatch
```

Comment:
Please document how it works. What is dispatched and where? Why only declare the `initializer_list{}` here?